web-platform/website/src/components/pages/account-settings/EditAccountEmailView.tsx

```diff
@@ -0,0 +1,278 @@
+
+      setValidationResult({
+        email: { error: emailErr },
+        password: { error: passwordErr },
+        passwordRules: [],
+        valid: false,
+      });
+
+      setVisibility(false)
+    }
+  }
+
+  return (
+    <Card className="flex-nowrap p-4">
+      {confirmationView ? (
+        <EmailChangedView email={emailRef.current} />
+      )
+        :
+        (
+          <Container
+            className={`${ContainerStyles.NoSpacing} ${styles.container} p-0`}
+          >
+            <Row className="mb-3">
+              <Col>
+                <h1 className={Fonts.Headline}>{strings.title}</h1>
+              </Col>
+            </Row>
+            <Row className="mb-3">
+              <Col>
+                <Form>
+                  <FormGroup className="mb-4">
+                    <Label for="email" className="fw-bold">
+                      {strings.newEmail}
+                    </Label>
+                    <Input
+                      id="email"
+                      type="email"
+                      name="email"
+                      autoFocus={true}
+                      value={email}
+                      onChange={handleChange}
+                      invalid={isEmailInvalid}
+                    />
+                    {isEmailInvalid && (
+                      <FormFeedback className="email-error">
+                        {validationResult?.email.error}
+                      </FormFeedback>
+                    )}
+                  </FormGroup>
+
+                  <FormGroup>
+                    <Label for="password" className="fw-bold">
+                      {strings.password}
+                    </Label>
+                    <div className="d-flex flex-row-reverse">
+                      <div className="w-100">
+                        <Input
+                          id="password"
+                          type={passwordVisible ? "text" : "password"}
+                          name="password"
+                          value={password}
+                          onChange={handleChange}
+                          invalid={isPasswordInvalid}
+                        />
+
+                        {isPasswordInvalid && (
+                          <FormFeedback className="password-error">
+                            {validationResult?.password.error}
+                          </FormFeedback>
+                        )}
+                      </div>
+
+
+                      {eyeIconVisible && (
+                        <img
+                          className="toggle-password align-self-center position-absolute m-1"
+                          src={!passwordVisible
+                            ? "img/icon-password-show.svg"
+                            : "img/icon-password-hide.svg"}
+                          alt={passwordVisible ? "open eye" : "eye with slash"}
+                          id="show-eye"
```

Comment:
We can remove the id if we are not using it and also use a css class and use Spacing enum instead of 24px (we have 24 px value there).

Reply:
True